ApplicationPattern.yaml

```diff
@@ -2191,6 +2310,7 @@ paths:
                 topology-operation-fc-update: '/v1/update-fc'
                 topology-operation-fc-port-update: '/v1/update-fc-port'
                 topology-operation-fc-port-deletion: '/v1/delete-fc-port'
+                topology-application-protocol: 'HTTP'
```

Comment:
uri definitions for callbacks of /v1/redirect-topology-change-information has protocol https which needs to be updated to reference topology-application-protocol {$request.body#topology-application-protocol}://...